package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-nimbus-clean",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

Comment:
Please do not bump the version here by hands. It will be automatically bumped after release base on commit message